app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
```

Comment:
Add a blank line line here and in the other places to follow PEP-8 import conventions (add a blank line between different groups of imports)